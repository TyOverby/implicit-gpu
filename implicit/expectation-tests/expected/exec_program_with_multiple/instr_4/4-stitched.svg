version https://git-lfs.github.com/spec/v1
oid sha256:58f6a783a8b4403400049ceca120c947fcdf09529e00735cb5f5de9c151b424c
size 1329
