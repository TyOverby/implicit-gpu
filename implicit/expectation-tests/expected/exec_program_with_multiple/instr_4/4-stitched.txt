1 line segments

Line Segment 0 
42 points
Clockwise? false
(10.0,1.0)
(11.0,0.99999905)
(12.0,1.0)
(13.0,1.0)
(14.0,1.0)
(15.0,1.0)
(15.0,2.0)
(15.0,3.0)
(15.0,4.0)
(15.0,5.0)
(15.0,6.0)
(15.0,7.0)
(15.0,8.0)
(15.0,9.0)
(15.0,10.0)
(15.000001,11.0)
(15.0,12.0)
(15.0,13.0)
(15.0,14.0)
(15.0,15.0)
(14.0,14.0)
(13.0,13.0)
(12.0,12.0)
(11.0,11.0)
(10.0,10.0)
(9.0,9.0)
(8.0,8.0)
(7.0,7.000001)
(6.0,6.0)
(5.0,5.000001)
(4.0,4.0)
(3.0,3.0)
(2.0,2.000001)
(1.0,1.0)
(2.0,1.0)
(3.0,1.0)
(4.0,1.0)
(5.0,1.0)
(6.0,0.9999995)
(7.0,1.0)
(8.0,1.0)
(9.0,1.0)
